packages/math-input/src/components/keypad-legacy/two-page-keypad.tsx

```diff
@@ -46,7 +50,8 @@ class TwoPageKeypad extends React.Component<Props> {
                 <Keypad style={[column, styles.keypad]}>
                     <Tabbar
                         items={["Numbers", "Operators"]}
-                        onSelect={(selectedItem) => {
+                        selectedItem={selectedPage}
+                        onSelectItem={(selectedItem) => {
                             this.setState({selectedPage: selectedItem});
                         }}
                     />
```

Comment:
We had duplicate state that I removed in this PR: Tabbar was managing what page it was one, but also components using Tabbar where managing which page they were on.

Instead I made Tabbar dumb and now we pass the selected page down to Tabbar.